hr_timesheet_no_closed_project_task/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "HR Timesheet No Closed Project-Task",
+    'summary': """
+        Prevent to select closed project or task on timesheet line""",
+    "version": "8.0.1.0.0",
+    "author": "ACSONE SA/NV,"
+              "Odoo Community Association (OCA)",
+    "website": "http://www.acsone.eu",
+    "category": "Human Resources",
+    "license": "AGPL-3",
+    "depends": [
+        "hr_timesheet_task",
+        "hr_timesheet_invoice",
+        "account_analytic_project",
+        "project_stage_closed",
```

Comment:
@dreispt the whole point of `project_stage_closed` is to have the `closed` flag on task types without the dependency on `sale_service`. The line you point to is there to avoid having the closed flag twice on the view in case sale_service is installed.